### DISCUSSION TITLE
Can I handle files in KFB format？

### DISCUSSION THREAD

**Discussion by ouronghuang** [libvips/libvips#3599] [General]

**Comment by lovell** [comment]
Does this refer to KFBIO e.g. https://www.kfbiopathology.com/digital-pathology/kf-pro-002/ ? If so, perhaps OpenSlide might be a better place to ask, as libvips can use that to open common slide formats.

**Reply by ouronghuang** [reply]
I want to use pyvips to directly process files in KFB format like SVS format

SVS format：

```
pyvips.Image.new_from_file('cell.svs', access='sequential')
```

So can this adapt to the KFB format？

```
pyvips.Image.new_from_file('cell.kfb', access='sequential')
```

I didn't see any dependencies in the [Optional Dependencies](https://github.com/libvips/libvips#optional-dependencies)

**Reply by jcupitt** [reply]
Hello, as Lovell says, this would be something for openslide:

https://github.com/openslide/openslide

You'll need to open an issue there and ask them to implement support.

You'll need to include full documentation of the file format, available under a free license. You will probably need to collaborate with the scanner manufacturers, and perhaps get some of their engineers to help you.
